.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        node: [12.x, 14.x]
+      fail-fast: false
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set node version to ${{ matrix.node }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Versions
+        run: yarn versions
+
+      - name: Install dependencies
+        run: yarn install #--frozen-lockfile
+
+      # - name: Check
+      #   run: yarn check
+
+      # - name: Lint
+      #   run: yarn lint
+
+      - name: Audit dependencies
+        run: yarn audit --groups dependencies
+
+      # - name: Audit devDependencies
+      #   run: yarn audit --groups devDependencies
+
+      # - name: Audit peerDependencies
+      #   run: yarn audit --groups peerDependencies
+
+      # - name: Audit optionalDependencies
+      #   run: yarn audit --groups optionalDependencies
+
+      - name: Build
+        run: yarn build
+
+      - name: Test
+        run: yarn test
```

Comment:
`test` are currently failing, cause the project is 2 years old (I think 😅)

Reply:
I believe I was working on fixing some tests when I stopped working on this project and the problem was something related to not being able to mock node's module resolution on the lower level I needed to mock it... but this is more about moving from travis to GH actions so I'm ok with ignoring failing tests for now.